services/movie_session.py

```diff
@@ -42,3 +41,41 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_movies(
+    genres_ids: list[int] = None,
+    actors_ids: list[int] = None,
+) -> QuerySet:
+    queryset = Movie.objects.all()
+
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+
+    return queryset
+
+
+def create_movie(
+    movie_title: str,
+    movie_description: str,
+    genres_ids: list = None,
+    actors_ids: list = None,
+) -> Movie:
+    movie = Movie.objects.create(
+        title=movie_title,
+        description=movie_description,
+    )
+    if genres_ids:
+        movie.genres.set(genres_ids)
+    if actors_ids:
+        movie.actors.set(actors_ids)
+
+    return movie
```

Comment:
The `get_movies` and `create_movie` functions are duplicated in this file. Consider removing these functions to avoid redundancy and maintain a single source of truth. These functions should be defined in the `movie.py` service file.